components/formats-api/src/loci/formats/in/DynamicMetadataOptions.java

```diff
@@ -516,9 +516,7 @@ public void loadOptions(String optionsFile, ArrayList<String> availableOptionKey
   public static String getMetadataOptionsFile(String id) {
     Location f = new Location(id);
     if (f != null && f.getParent() != null) {
-      String p = f.getParent();
-      String n = f.getName();
-      return new Location(p, n + ".bfoptions").getAbsolutePath();
+      return new Location(id + ".bfoptions").getAbsolutePath();
```

Comment:
Can you explain what behavior you are trying to fix on Windows here?

Reply:
``\t1.tiff`` on windows will be turned into ``\\t1.tiff.bfoptions`` instead of ``D:\t1.tiff.bfoptions``. 
The case ``\foo\\t1.tiff`` works as expected without the change

Reply:
Immediate thought is that `\foo\t1.tiff` is not a valid Windows path anyways no?  The alternate should be to change the unit test data provider itself to work on both platforms.

Reply:
The tests are only for linux so I went for the approach of replacing ``/`` by ``\`` as I had to do for the other sets of tests.
I can revisit the tests
What was the rational behind the split in the first place?

Reply:
``\foo\t1.tiff`` will be valid from the current drive

Reply:
@dgault: can you comment on the use of `new Location(String, String)` versus `new Location(String)`. I seem to remember a few discussions around this but only found https://github.com/ome/bioformats/pull/2914